metrics/prometheus-claim.yaml

```diff
@@ -2,11 +2,11 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: prometheus-db
-  namespace: frontdoor
+  name: prometheus-data
+  namespace: cockpit
 spec:
   accessModes:
   - ReadWriteOnce
   resources:
     requests:
-      storage: 10Gi
+      storage: 5Gi
```

Comment:
This change is delibarate ? (Not included in the commit msg or wondering if it's missed to explain, accident or just trivial)

Reply:
It is deliberate, see the [onboarding request](https://pagure.io/centos-infra/issue/991#comment-831950). I adjusted the commit message accordingly, thanks for pointing out!